ios/RN/RNAssetWriter.m

```diff
@@ -0,0 +1,330 @@
+//
+//  RNAssetWriter.m
+//  RNCamera
+//
+//  Created by Marius Reimer on 10/6/18.
+//
+
+#import "RNAssetWriter.h"
+
+@implementation RNAssetWriter
+
+{
+    BOOL dualModeEnabled;
+    BOOL bFirstAudio;
+    BOOL bWriting;
+    
+    int64_t frameNumber;
+    size_t bytesWritten;
+    
+    CMTime startTime;
+    CMTime previousFrameTime;
+    CMTime previousAudioTime;
+    CMTime firstAudioTimeStamp;
+    CMTime firstVideoTimestamp;
+    
+    NSDate* initDate;
+    
+    CMSampleBufferRef firstAudioBuffer;
+    
+    AVAssetWriter* assetWriter;
+    AVAssetWriterInput* assetVideoWriterInput;
+    AVAssetWriterInput* assetAudioWriterInput;
+    AVAssetWriterInputPixelBufferAdaptor* pixelBufferAdaptor;
+    
+    dispatch_queue_t writerQueue;
+}
+
+- (BOOL)initRecording {
+    writerQueue = dispatch_queue_create("com.mariusreimer.writerQueue", dispatch_queue_attr_make_with_qos_class(DISPATCH_QUEUE_SERIAL, QOS_CLASS_USER_INTERACTIVE, 0));
```

Comment:
Hello, I noticed a couple of things that are not good for an open source project, like this label.